### PR TITLE
Add Temel and Completion packages.

### PR DESCRIPTION
**Pull request by f**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +127,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 ## Template Engine
   * [Bunny](https://github.com/samsheff/Bunny) - A simple HTML templating language for Crystal, same syntax as erb
   * [crustache](https://github.com/MakeNowJust/crustache) - [{{Mustache}}](https://mustache.github.io) for Crystal
+  * [temel](http://github.com/f/temel) Extensible HTML::Builder alternative for Crystal, supports custom tag definitions
```
Comment (veelenga) [inline README.md]: * Has to be `https` (not `http`)
* Missing a dash between a link and description

**Comment by veelenga** [comment]
Thanks for contributing. Please update and do git squash/amend.

**Comment by f** [comment]
Thank you! Updated and amend'ed!

**Comment by veelenga** [comment]
Thanks :+1:

**Pull request merged**